report/4.Design/HardwareTest.tex

```diff
@@ -0,0 +1,24 @@
+\section{Hardware testing}
+To ensure the most accurate results, it is important to ensure the quality and precision of the chosen hardware to be at its highest.
+Therefore, a hardware test has been performed on the motors, checking their precision and quality.
+
+\subsection*{Motors}
+To ensure motor capabilities, the testing has to try out combinations of target angles and movement speed, and to what extend precision is reached, considering the braking distance.
+Therefore a setup combining an angle, and a velocity ranging 0-100 is setup as seen in appendix \ref{appendix:motortest}.
+
+The terms: Stable, Small Correction and Huge Correction is used, to specify the stopping at the angle, and to elaborate on, not only precision, but the amount of correction needed, to stop at the exact angle specified.
```

Comment:
are used

Reply:
btw skal de stå med store?

Reply:
```suggestion
The terms stable, small correction and huge correction are used to specify the stopping at the angle, and to elaborate on, not only precision, but the amount of correction needed, to stop at the exact angle specified.
```

jeg forstår ikke "to specity the stopping at the angle" sætningen, er jeg retarderet???????